src/panels/config/lovelace/resources/ha-config-lovelace-resources.ts

```diff
@@ -0,0 +1,184 @@
+import { showConfirmationDialog } from "../../../../dialogs/generic/show-dialog-box";
+import { lovelaceTabs } from "../ha-config-lovelace";
+import { loadLovelaceResources } from "../../../lovelace/common/load-resources";
+
+@customElement("ha-config-lovelace-resources")
+export class HaConfigLovelaceRescources extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public isWide!: boolean;
+  @property() public narrow!: boolean;
+  @property() public route!: Route;
+  @property() public showAdvanced!: boolean;
+  @property() private _resources: LovelaceResource[] = [];
+
+  private _columns = memoize(
+    (_language): DataTableColumnContainer => {
+      return {
+        url: {
+          title: this.hass.localize(
+            "ui.panel.config.lovelace.resources.picker.headers.url"
+          ),
+          sortable: true,
+          filterable: true,
+          direction: "asc",
+        },
+        type: {
+          title: this.hass.localize(
+            "ui.panel.config.lovelace.resources.picker.headers.type"
+          ),
+          sortable: true,
+          filterable: true,
+          template: (type) =>
+            html`
+              ${this.hass.localize(
+                `ui.panel.config.lovelace.resources.types.${type}`
+              ) || type}
+            `,
+        },
+      };
+    }
+  );
+
+  protected render(): TemplateResult {
+    if (!this.hass || this._resources === undefined) {
+      return html`
+        <hass-loading-screen></hass-loading-screen>
+      `;
+    }
+
+    return html`
+      <hass-tabs-subpage-data-table
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        back-path="/config"
+        .route=${this.route}
+        .tabs=${lovelaceTabs}
+        .columns=${this._columns(this.hass.language)}
+        .data=${this._resources}
+        .showAdvanced=${this.showAdvanced}
+        @row-click=${this._openDialog}
+      >
+      </hass-tabs-subpage-data-table>
+      <ha-fab
+        ?is-wide=${this.isWide}
+        ?narrow=${this.narrow}
+        icon="hass:plus"
+        title="${this.hass.localize(
+          "ui.panel.config.lovelace.resources.picker.add_resource"
+        )}"
+        @click=${this._openDialog}
+      ></ha-fab>
+    `;
+  }
+
+  protected firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
+    this._getResources();
+  }
+
+  private async _getResources() {
+    this._resources = await fetchResources(this.hass.connection);
+  }
+
+  private async _openDialog(ev: CustomEvent): Promise<void> {
+    const id = (ev.detail as RowClickedEvent).id;
+    const resource = this._resources.find((res) => res.id === id);
+    showResourceDetailDialog(this, {
+      resource,
+      createResource: async (values) => {
+        const created = await createResource(this.hass!, values);
+        this._resources = this._resources!.concat(created).sort((res1, res2) =>
+          compare(res1.url, res2.url)
```

Comment:
We should show panels that have a sidebar first, sorted by name, then other panels sorted by url.